recipes/prepare.rb

```diff
@@ -16,6 +16,12 @@
   action :sync
 end
 
+git "#{build_user_home}\\integrations-core" do
+  repository 'https://github.com/DataDog/integrations-core'
+  revision default['dd-agent-builder']['dd-agent_branch']
```

Comment:
So we'll have a strict correspondence between the `dd-agent` and `integrations-core` branch? I think even if it's the case, it would be nice to be able to use different branches for testing purposes (or build beta/specific releases). What do you think?